PurchasesCoreSwift/Logging/Strings/PurchaserInfoStrings.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  PurchaserInfoStrings.swift
+//  PurchasesCoreSwift
+//
+//  Created by Tina Nguyen on 12/11/20.
+//  Copyright © 2020 Purchases. All rights reserved.
+//
+
+import Foundation
+
+@objc(RCPurchaserInfoStrings) public class PurchaserInfoStrings: NSObject {
+    @objc public var checking_intro_eligibility_error: String { "couldn't check intro eligibility locally, error: %@" } //error
+    @objc public var checking_intro_eligibility_result: String { "Local intro eligibility computed locally. Result: %@" } //debug
```

Comment:
the `local` bit is kinda important here, since it means the difference between hitting the server or not...
how about 
`checking_intro_eligibility_locally_error` and `checking_intro_eligibility_locally_result`